### ISSUE TITLE
Not supporting certain backlight values causes a crash

### ISSUE THREAD

**Issue by MxBlu** [the-via/keyboards#23]
Not sure where you want issues, so I'll open them here for now.

When trying to make some of the RGB code work, I found that not support backlight values 7 and 8 (disable on USB suspend and disable after timeout) causes a crash when entering the Advanced menu in Lighting.

I reproduced this on MxSS with this extension over the PR:
``` json
"lighting": {
    "extends": "qmk_underglow",
    "supportedBacklightValues": [
      9,
      10,
      11,
      12,
      14,
      16,
      18,
      20
    ]
  },
Adding 7 and 8 fixed the crash.

**Comment by olivia** [comment]
@Wilba6582 should we only expect those commands to work on WT lighting?

**Comment by Wilba6582** [comment]
```	id_disable_when_usb_suspended = 0x07,
	id_disable_after_timeout = 0x08,```

They might be valid for other implementations.

Then again, there is no reason why all lighting implementations have to use the same set of values. They were defined for `wt_rgb_backlight` and `wt_mono_backlight` specifically.

**Comment by MxBlu** [comment]
QMK supports these but through buildtime flags, I think they're hidden somewhere inside tmk. Anyhow, I'd agree with Wilba, no reason to disable it outright. Just would be nice to not have the UI crash.

Keyboards defining their own set of values would be neat, but the UI and handling would need to extend that: i.e. id_disable_after_timeout is a boolean, handle as a switch. CAPS_LOCK_INDICATOR_COLOR is a color, handle as a color palette, etc.
Then there's the thought of which values are valid for what modes/settings. I dunno how to resolve that, just food for thought.

**Comment by olivia** [comment]
> ```
> id_disable_when_usb_suspended = 0x07,
> id_disable_after_timeout = 0x08,
> ```
> 
> They might be valid for other implementations.
> 
> Then again, there is no reason why all lighting implementations have to use the same set of values. They were defined for `wt_rgb_backlight` and `wt_mono_backlight` specifically.

What I'm speaking to specifically is cases where the values represent the same functionality i.e. there shouldn't be 2 different values that represent disabling lighting when USB is disconnected. We should be keeping the API as lean as possible.

**Comment by MxBlu** [comment]
The core issue behind this PR was resolved with 1.2.4, so closing this issue.
